proposals/NNNN-vk-sampled-texture.md

```diff
@@ -0,0 +1,174 @@
+<!-- {% raw %} -->
+
+# Vulkan Sampled Textures
+
+* Proposal: [NNNN](NNNN-vk-sampled-texture.md)
+* Author(s): [Cassandra Beckley](https://github.com/cassiebeckley)
+* Sponsor: [Steven Perron](https://github.com/s-perron) and TBD
+* Status: **Under Consideration**
+* Required Version: HLSL 202x
+<!--
+* PRs: [#NNNN](https://github.com/microsoft/DirectXShaderCompiler/pull/NNNN)
+* Issues:
+  [#NNNN](https://github.com/microsoft/DirectXShaderCompiler/issues/NNNN)
+-->
+
+## Introduction
+
+We propose a new set of `vk::SampledTexture*` texture types representing
+Vulkan's combined image sampler types (`OpTypeSampledImage`).
+
+## Motivation
+
+The [existing annotation](https://github.com/microsoft/DirectXShaderCompiler/wiki/Vulkan-combined-image-sampler-type)
+for combined image samplers (`[[vk::combinedImageSampler]]`) was designed with
+the explicit goal of avoiding defining new HLSL types and functions. This was
+intended to minimize the number of changes to the frontend needed to implement
+the feature. However, it is verbose and confusing for users, and we do not
+intend to implement it in upstream Clang.
```

Comment:
I would mention that the implementation requires an transformation to run that pairs up textures and samplers, but it cannot work in all cases. This can lead to very subtle errors when, for example, the same sampler is used with two different textures.